lmfit/models.py

```diff
@@ -390,7 +390,7 @@ class SplitLorentzianModel(Model):
 
     .. math::
 
-        f(x; A, \mu, \sigma, \sigma_r) = \frac{2 A}{\pi (\sigma+\sigma_r)} \big[\frac{1}{(x - \mu)^2 + \sigma^2} * H(\mu-x) + \frac{1}{(x - \mu)^2 + \sigma_r^2} * H(x-\mu)\big]
+        f(x; A, \mu, \sigma, \sigma_r) = \frac{2 A}{\pi (\sigma+\sigma_r)} \big[\frac{\sigma^2}{(x - \mu)^2 + \sigma^2} * H(\mu-x) + \frac{\sigma_r^2}{(x - \mu)^2 + \sigma_r^2} * H(x-\mu)\big]
```

Comment:
the message for this commit could/should include: ```Fixes #584```

Reply:
This required a rebase-edit-history, but should be OK

Reply:
I don't think that works... to close an Issue automatically when a commit referencing it gets merged it should follow some [rules](https://help.github.com/en/articles/closing-issues-using-keywords). That's why I suggested the exact syntax and preferably it shouldn't be part of the title but separately after a newline.

Reply:
Oh, maybe I misunderstood the request.FWIW, I'm OK with changing the commit history...

Reply:
changing the commit history should never be done on branch (i.e., ```master```) in the main repository, but for PRs I don't see any issue. Often that is the only way to actually keep a clean/organized history when addressing comments and such. 

The reason I was suggesting to add this statement to the commit message and close the issue automatically, is that it keeps Issues and PRs connected which is nice. But then again, once can always close the issue afterwards with adding the commit (or PR) that fixes it - so let's not overthink it. I am fine with it - the content is great anyway: it fixes an issue and improves the test coverage quite significantly.